hoomd_validation/config_parser.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, config_file_path=DEFAULT_CONFIG_PATH):
             config.get("max_cores_submission", 16))
         self["max_gpus_submission"] = int(config.get("max_gpus_submission", 1))
         self["max_walltime"] = int(config.get("max_walltime", 24))
```

Comment:
```suggestion
        self["max_walltime"] = float(config.get("max_walltime", 24))
```
For consistency with the below change.